### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AbhiTaker**
the updated form of readme.md which helps in better understanding of the working of project

### CODE REVIEW

**Review by mrsaicharan1 — CHANGES_REQUESTED** [review]
Please re-write without any grammatical mistakes. Take care of sentence formation.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+* run `python p1` command (Linux users should make the script executable first)
+* now you can query definition of words in the popped up CLI
+
+# How it works
+
+* User queries the definition of word in CLI
+* word's meaning is displayed if it exists in database 
+* otherwise it searches for the closest match of the queried word
+* if closest match found by program is the required word, it prints word's meaning
+* appropriate message is displayed if the required word doesn't exists in database
```
Comment (mrsaicharan1) [inline README.md]: exist*

**Comment by AbhiTaker** [comment]
corrected the spelling

**Comment by mrsaicharan1** [comment]
Shows some conflicts. Please fix. @AbhiTaker

**Comment by AbhiTaker** [comment]
conflicts resolved

**Pull request merged**